### PR TITLE
fix more deprecations and errors on 0.7

### PR DESCRIPTION
**Pull request by rdeits**
@blandry this is a PR against your branch from https://github.com/JuliaRobotics/MechanismGeometries.jl/pull/5 to try to fix up some more test errors and deprecation warnings.

### CODE REVIEW

**Comment by rdeits** [comment]
We might need to wait for new package tags including https://github.com/JuliaGeometry/GeometryTypes.jl/pull/137 and https://github.com/JuliaIO/MeshIO.jl/pull/41

File: .travis.yml [hunk 1]
```diff
@@ -12,9 +13,9 @@ git:
 
 # uncomment the following lines to allow failures on nightly julia
 # (tests will run but not make your overall status red)
-matrix:
- allow_failures:
- - julia: nightly
+# matrix:
+#  allow_failures:
+#  - julia: nightly
```
Comment (tkoolen) [inline .travis.yml:18]: I don't think this is a good idea.

File: src/MechanismGeometries.jl [hunk 2]
```diff
@@ -1,17 +1,18 @@
-__precompile__(true)
+VERSION < v"0.7.0-beta2.199" && __precompile__()
```
Comment (tkoolen) [inline src/MechanismGeometries.jl:1]: This should (luckily) no longer be necessary on 0.7 (i.e., you can just leave the `__precompile__(true)`). Ref https://github.com/JuliaLang/julia/pull/28459.

**Comment by tkoolen** [comment]
For the sake of expedience given JuliaCon, should this just be a PR against JuliaRobotics/MechanismGeometries?

**Comment by rdeits** [comment]
Sure, that sounds good. I'm waiting on the GeometryTypes/MeshIO updates anyway, so I'll pick this up again tomorrow

**Comment by blandry** [comment]
Awesome thanks guys. Sorry I'm new to these packages so still finding my way around. If that's relevant at all I also had to add a "using Dates" to InteractBase.jl to get basic scripts to work.

**Pull request merged**